sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/util/configuration/ConfigurationBuilder.java

```diff
@@ -37,8 +38,8 @@ public ConfigurationBuilder() {
     /**
      * Creates {@code ConfigurationBuilder} with configuration source.
      *
-     * <!-- src_embed com.azure.core.util.configuration.Configuration -->
-     * <!-- end com.com.azure.core.util.configuration.Configuration -->
+     * <!-- src_embed com.azure.core.util.Configuration -->
+     * <!-- end com.com.azure.core.util.Configuration -->
```

Comment:
nit;
```suggestion
     * <!-- src_embed com.generic.core.util.configuration.Configuration -->
     * <!-- end com.com.generic.core.util.configuration.Configuration -->
```

Reply:
these should be done eventually before the demo. We need to make sure we don't have any references with azure in it.